packages/wrangler/src/git-client.ts

```diff
@@ -12,10 +16,19 @@ export async function isInsideGitRepo(cwd: string) {
 
 /**
  * Check whether git is installed by trying to run it.
+ *
+ * Returns the installed version if git is installed, and false otherwise
  */
 export async function isGitInstalled() {
```

Comment:
NIT: now that this function is not just checking for the existing of git, I think this function name is misleading.
How about:

```suggestion
export async function getGitVersion(): string|null {
```

where `null` indicates that git is not installed?